src/api/trace.zig

```diff
@@ -1,9 +1,91 @@
+const std = @import("std");
+
 pub const Tracer = @import("trace/tracer.zig").Tracer;
 pub const TracerProvider = @import("trace/provider.zig").TracerProvider;
 pub const TracerConfig = @import("trace/config.zig").TracerConfig;
 
+pub const TraceID = struct {
+    value: [16]u8,
+
+    const Self = @This();
+
+    pub fn init(value: [16]u8) Self {
+        return .{
+            .value = value,
+        };
+    }
+
+    pub fn isValid(self: Self) bool {
+        return Self.isValidValue(self.value);
+    }
+
+    pub fn isValidValue(value: [16]u8) bool {
+        for (value) |item| {
+            if (item != 0) {
+                return true;
+            }
+        }
```

Comment:
Q: why do we need two functions?
If it's part of the  OTel specification, can you add a comment referencing the specification link that mandates having them?

Reply:
`isValidValue()` is just needed because the check function i need before transforming the raw array to a trace id.
so I will remove it, thanks!